core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.html

```diff
@@ -142,21 +142,29 @@
       <i class="material-icons"><[isPlayingUploadedAudio() ? 'pause' : 'play_arrow']></i>
     </button>
   </div>
-  <div class="oppia-slider-section oppia-content-wrapper">
+  <div class="oppia-slider-section oppia-content-wrapper fx-main-center fx-cross-center full-height">
     <div ng-if="audioLoadingIndicatorIsShown">
       <md-progress-linear md-mode="indeterminate"></md-progress-linear>
     </div>
-    <div ng-if="!audioLoadingIndicatorIsShown">
-      <md-slider ng-model="track.progress" ng-model-options="{getterSetter: true}" aria-label="audio-slider">
-      </md-slider>
+    <div ng-if="!audioLoadingIndicatorIsShown" class="fx-row fx-main-center fx-cross-center">
+      <oppia-audio-slider class="full-width"
+                          [value]="AudioPlayerService.getCurrentTime()"
+                          [max]="durationSecs"
+                          (value-change)="setProgress($event)"
+                          [thumb-label]="true"
+                          aria-label="audio-slider">
+      </oppia-audio-slider>
     </div>
   </div>
   <div class="oppia-content-wrapper oppia-recording-timer">
     <div ng-if="isAudioAvailable && audioIsLoading">
       <[0 | formatTimer]> / <[durationSecs | formatTimer]>
     </div>
     <div ng-if="audioTimerIsShown && !audioIsLoading">
-      <[getUploadedAudioTimer()]>
+      <[AudioPlayerService.getCurrentTime() | formatTimer]> / <[durationSecs | formatTimer]>
+    </div>
+    <div ng-if="!audioTimerIsShown && !audioIsLoading">
+       -- / --
```

Comment:
Are we going to present this even when `audioTimerIsShown` is false? (Do we had the same behavior before?)